packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -561,7 +554,7 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
 
     protected abstract getInitialSnapshotDetails(): Promise<ISnapshotDetails>;
 
-    protected abstract getInitialGCDetails(): Promise<IGCDetails>;
+    protected abstract getInitialGCDetails(): Promise<IGCDetails | undefined>;
```

Comment:
I do not know if it makes sense, but I think you use IGCDetails | undefined everywhere. Does it make sense to have a type for it?

Reply:
Removed undefined from this.